lib/recording/RecordingEntrySession.js

```diff
@@ -484,6 +484,24 @@ class RecordingEntrySession {
         this.logger = loggerBase.getLogger('RecordingEntrySession', '[' + this.recordingSessionId + '] ');
         return this.playlistGenerator.setBasePath(this.recordingSessionPath);
     }
+
+    is24By7RecordingEnabled() {
```

Comment:
please rename to isContinuousRecordingEnabled

Reply:
done